.husky/pre-commit

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 . "$(dirname $0)/_/husky.sh"
 
-yarn run tsc -p tsconfig.json --noEmit
```

Comment:
[F] This now throws some wacky errors

Reply:
Can safely replace with the command on the right or drop altogether. Makes me think we no longer need the `tsconfig.json` at the root of the monorepo.